<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CCompoundsManagerClass</class>
  <widget class="QDialog" name="CCompoundsManagerClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>512</width>
        <height>320</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Used compounds</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <property name="autoFillBackground">
      <bool>false</bool>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
        <layout class="QVBoxLayout" name="verticalLayout">
          <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                    <widget class="QLabel" name="label_7">
                      <property name="text">
                        <string>Active compounds</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QTableWidget" name="usedCompounds">
                      <property name="editTriggers">
                        <set>QAbstractItemView::NoEditTriggers</set>
                      </property>
                      <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                      </property>
                      <property name="selectionBehavior">
                        <enum>QAbstractItemView::SelectRows</enum>
                      </property>
                      <attribute name="horizontalHeaderVisible">
                        <bool>false</bool>
                      </attribute>
                      <attribute name="horizontalHeaderStretchLastSection">
                        <bool>true</bool>
                      </attribute>
                      <property name="toolTip">
                        <string>Active compounds</string>
                      </property>
                      
                      <property name="whatsThis">
                        <string>List of active compounds added to the flowsheet</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                    <spacer name="verticalSpacer_2">
                      <property name="orientation">
                        <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                        <size>
                          <width>0</width>
                          <height>0</height>
                        </size>
                      </property>
                    </spacer>
                  </item>
                  <item>
                    <widget class="QPushButton" name="addSelectedCompound">
                      <property name="text">
                        <string>&lt;--</string>
                      </property>
                      <property name="toolTip">
                        <string>Add to active compounds</string>
                      </property>
                      
                      <property name="whatsThis">
                        <string>Add the selected compound to the list of active compounds of the flowsheet</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="verticalSpacer">
                      <property name="orientation">
                        <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                        <size>
                          <width>0</width>
                          <height>0</height>
                        </size>
                      </property>
                    </spacer>
                  </item>
                  <item>
                    <widget class="QPushButton" name="removeSelectedCompound">
                      <property name="text">
                        <string>X</string>
                      </property>
                      <property name="toolTip">
                        <string>Remove from active compounds</string>
                      </property>
                      
                      <property name="whatsThis">
                        <string>Remove the selected compound from the list of active compounds of the flowsheet</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="verticalSpacer_3">
                      <property name="orientation">
                        <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                        <size>
                          <width>0</width>
                          <height>0</height>
                        </size>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                    <widget class="QLabel" name="label_8">
                      <property name="text">
                        <string>Available compounds</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QTableWidget" name="availableCompounds">
                      <property name="editTriggers">
                        <set>QAbstractItemView::NoEditTriggers</set>
                      </property>
                      <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                      </property>
                      <property name="selectionBehavior">
                        <enum>QAbstractItemView::SelectRows</enum>
                      </property>
                      <attribute name="horizontalHeaderVisible">
                        <bool>false</bool>
                      </attribute>
                      <attribute name="horizontalHeaderStretchLastSection">
                        <bool>true</bool>
                      </attribute>
                      <property name="toolTip">
                        <string>Available compounds</string>
                      </property>
                      
                      <property name="whatsThis">
                        <string>List of available compounds from materials database</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
                <spacer name="horizontalSpacer">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                    <size>
                      <width>40</width>
                      <height>20</height>
                    </size>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="pushButtonOk">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>OK</string>
                  </property>
                  <property name="default">
                    <bool>false</bool>
                  </property>
                  <property name="flat">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="pushButtonCancel">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Cancel</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <tabstops>
    <tabstop>usedCompounds</tabstop>
    <tabstop>availableCompounds</tabstop>
    <tabstop>addSelectedCompound</tabstop>
    <tabstop>removeSelectedCompound</tabstop>
    <tabstop>pushButtonOk</tabstop>
    <tabstop>pushButtonCancel</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>